pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerConfiguration.java

```diff
@@ -321,4 +322,21 @@ public ConsumerConfiguration setProperties(Map<String, String> properties) {
     public Map<String, String> getProperties() {
         return properties;
     }
+    
+     /** 
+     * @param initializeSubscriptionOnLatest the initializeSubscriptionOnLatest to set
+     * Set cursor position when subscribing to the topic first time
+     * <p>
+     * Default is {@value true} which means {@link MessageId.lastest}
```

Comment:
It's a bit unclear from this API where the subscription will be initialized if it's not on latest. We should make it clear that it's either "latest" or "earliest" (with latest as default). 

Again, I think that an enum would be a better option here to better convey the 2 options.